### PR TITLE
(fix) [SD-3630] "place" drop down list is empty in the user profile section

### PR DESCRIPTION
**Pull request by sivakuna-aap**
Resolves [SD-3630](https://dev.sourcefabric.org/browse/SD-3630)

### CODE REVIEW

**Comment by plamut** [comment]
Should we add a regression e2e test to cover the issue that has now been fixed?

Looks good though. :+1:

File: client/app/scripts/superdesk-users/views/user-preferences.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
                 <div class="legend" translate>Article Defaults</div>
 
                 <div class="legend" translate>Dateline</div>
-                <div sd-info-item>
+                <div sd-info-item ng-if="preferencesLoaded === true">
```
Comment (petrjasek) [inline client/app/scripts/superdesk-users/views/user-preferences.html]: this `=== true` seems extra, it's only `true/false` right?

**Comment by petrjasek** [comment]
:+1:

File: client/app/scripts/superdesk-users/views/user-preferences.html (hunk 1 above)
Comment (sivakuna-aap) [inline client/app/scripts/superdesk-users/views/user-preferences.html]: Agreed. But I wanted to be explicit here as even a non-null value evaluates to true.

Comment (petrjasek) [inline client/app/scripts/superdesk-users/views/user-preferences.html]: I see, explicit is good

**Pull request merged**